README.md

```diff
@@ -414,21 +421,28 @@ cd client; go test  -tags sqlite  .
 
 We use [Cypress](https://www.cypress.io) to run our e2e tests. You can run all tests using:
 
-```shell script
+<pre type="make/command">
 make test-e2e
-```
+</pre>
 
 If you intend developing e2e tests, run the following command for more details:
 
-```shell script
+<pre type="repo/executable">
 ./test/e2e/run.sh
-```
-
+</pre>
```

Comment:
I tag this block as containing an executable that exists in this repository. Text-Runner verifies that this file exists and is executable. It doesn't run it. I assume you have separate tests for it. 

We could run it here, then we would have to tag it to be of type `shell/command`.